cmd/resource_cmds.go

```diff
@@ -0,0 +1,39 @@
+// this file WILL be generated (sc-241153)
+
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"ldcli/cmd/resources"
+)
+
+func addAllResourceCmds(rootCmd *cobra.Command, client *http.Client) {
+	// Resource commands
+	gen_TeamsResourceCmd := resources.NewResourceCmd(
+		rootCmd,
+		"teams",
+		"A team is a group of members in your LaunchDarkly account.",
```

Comment:
This is the first sentence from the "Teams" tag description (after the headers). 

Our previously established format is "Make requests (list, create, etc.) on {resource name}" - not sure which is better in this context 🤔